### PR TITLE
Make tox.ini consistent with the project.

### PR DESCRIPTION
**Pull request by ccamacho**
This commit includes the tox.ini file
at the root of the project because it is not
exclusive of Github actions.
This commit makes sure all the tox.ini environments are passing (while are showing lint errors) for future fixes. This commit makes sure the linters check verify all the linter commands from a common tox environment.

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    bash
+commands =
+    bash -c "echo '==> Installing Event Driven Ansible...'"
+    bash -c "ansible-galaxy collection build -v --force --output-path releases/ && \
+             ansible-galaxy collection install --force --force-with-deps releases/ansible-eda-$(cat ./galaxy.yml | shyaml get-value version).tar.gz"
+
+[testenv:ruff]
+deps = {[testenv:linters]deps}
+commands =
+    bash -c 'ruff check --select ALL --ignore INP001 extensions/eda/plugins || true'
```
Comment (ccamacho) [inline tox.ini:29]: The idea here is to push additional PRs removing the || true once each lint check is passing.

**Comment by ccamacho** [comment]
@Alex-Izquierdo can you please enable the GH actions run here? Thanks!

**Comment by Alex-Izquierdo** [comment]
Hi @ccamacho The PR has conflicts to resolve

**Comment by Alex-Izquierdo** [comment]
Replaced by https://github.com/ansible/event-driven-ansible/pull/158

**Pull request closed without merging**